pysoa/test/stub_service.py

```diff
@@ -370,6 +371,7 @@ def wrapped_send_request(client, service_name, actions, *args, **kwargs):
                     request_id,
                     continue_on_error,
                 )
+                self._stub_action_unwrapped_call_id_map[request_id] = unwrapped_request_id
```

Comment:
You don't need these two lines with `_stub_action_unwrapped_call_id_map` anymore.